package.nix

```diff
@@ -7,7 +7,7 @@
 , boehmgc
 , nlohmann_json
 , bison
-, boost
+, boost-nix
```

Comment:
I think this file should stay as-is, and we should instead have `boost = final.boost-nix;` in the `flake.nix`.